### PR TITLE
Tabs Style Change

### PR DESCRIPTION
**Pull request by rtkaaho**
- Modifies the look of the tabs
- Removes ink bar flickering when the component first renders

Screen Shot 2020-04-14 at 11 12 59 AM

### CODE REVIEW

File: src/components/tabs/Tabs.tsx [hunk 1]
```diff
@@ -101,15 +101,16 @@ export const Tabs: TabsFunctionComponent<TabsProps> = ({
...
-            }}
-          />
-        )}
+        {/* {activeItem == null ? null : ( */}
+        <Inkbar
+          theme={theme}
+          style={{
+            visibility: activeItem == null ? 'hidden' : 'visible',
+            width: activeItem == null ? 'unset' : activeItem.width,
+            left: activeItem == null ? 'unset' : activeItem.x,
```
Comment (rtkaaho) [inline src/components/tabs/Tabs.tsx:110]: This removes the flickering that occurs when the component first renders due to the fact that the `activeItem` remains unset until after the selected tabs ref is fully qualified.

File: src/components/typography/Body.tsx [hunk 2]
```diff
@@ -34,22 +37,24 @@ const StyledBody = styled.div<StyledBodyProps>`
   `}
 `;
 
-export const Body: React.FunctionComponent<BodyProps> = ({
-  children,
-  className,
-  disabled,
-}) => {
+export const Body: React.FunctionComponent<BodyProps> = React.forwardRef<
```
Comment (rtktpeng) [inline src/components/typography/Body.tsx:40]: Disclaimer: I just got introduced to `forwardRef` by reading up related docs, source code and type definitions.

1. `React.forwardRef` has a detailed returned type on its own, and I don't think we need to override it with `React.FunctionComponent<BodyProps>`.
```
function forwardRef<T, P = {}>(Component: RefForwardingComponent<T, P>): ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>;
```

2. I think it's more _technically correct_ to use an interface without the property `ref` as the second generic type argument of `forwardRef`. The need for `ref` is "injected" by `forwardRef`.
In other words, the expected typing behaviour is: 
`props.ref` should not exist, since `ref` is provided as the second argument in `(props, ref) => {}`. 
To achieve this, I would specify the generic arguments like 
```
const Body = React.forwardRef<
  HTMLDivElement,
  Omit<BodyProps, 'ref'>
>
```

Comment (rtkaaho) [reply src/components/typography/Body.tsx:40]: In this case its one or the other. Either I add to the interface the `ref` or `children`. The Type specifying `React.FunctionComponent` gives me the `children` prop typings for free.

Comment (rtkaaho) [reply src/components/typography/Body.tsx:40]: What I like about adding the `ref` to the interface is I get to specify to the user exactly what kind of element they will be getting back. I am doing this in other components. So for now I am going to leave it as is. If we start running into issues I will know the first place to look :)
